src/main/asciidoc/index.adoc

```diff
@@ -170,73 +178,10 @@ Each event contains a Json Object with:
 
 == The half-open state
 
-When the circuit is "open", calls to the circuit breaker fail immediately, without any attempt to execute the real
-operation. After a suitable amount of time (configured from
-{@link io.vertx.circuitbreaker.CircuitBreakerOptions#setResetTimeout(long)}, the circuit breaker decides that the
+When the circuit breaker is `open`, calls to the circuit breaker fail immediately, without any attempt to execute the real
+operation. After a suitable amount of time (configured by
+{@link io.vertx.circuitbreaker.CircuitBreakerOptions#setResetTimeout(long)}), the circuit breaker decides that the
 operation has a chance of succeeding, so it goes into the `half-open` state. In this state, the next call to the
-circuit breaker is allowed to execute the dangerous operation. Should the call succeed, the circuit breaker resets
+circuit breaker is allowed to execute the guarded operation. Should the call succeed, the circuit breaker resets
 and returns to the `closed` state, ready for more routine operation. If this trial call fails, however, the circuit
 breaker returns to the `open` state until another timeout elapses.
-
-== Reported exceptions
-
-The fallback receives a:
-
-* {@link io.vertx.circuitbreaker.OpenCircuitException} when the circuit breaker is opened
```

Comment:
Why is this section about reported exception deleted ?

Reply:
It isn't deleted, it's moved up a little: https://github.com/vert-x3/vertx-circuit-breaker/pull/71/commits/41fd1cfaa6fcf964305be1e4dcd1bc000df487a9#diff-97965d9bb2e3f738e48945f14914e26a4f3c5ae77cfeb5f339009e191124f2beR88